warehouse/upload.go

```diff
@@ -1615,7 +1630,7 @@ func (job *UploadJobT) setUploadError(statusError error, state string) (string,
 
 	// Reset the state as aborted if max retries
 	// exceeded.
-	uploadErrorAttempts := 5
+	uploadErrorAttempts := uploadErrors[state]["attempt"].(int)
```

Comment:
This got pushed by mistake https://github.com/rudderlabs/rudder-server/pull/2956